lib/rabbit_feed/consumer.rb

```diff
@@ -7,7 +7,9 @@ module Consumer
     def run
       ConsumerConnection.instance.consume do |raw_event|
         event = Event.deserialize raw_event
+        RabbitFeed.log.info {{ event: :message_received, metadata: event.metadata }}
         event_routing.handle_event event
+        RabbitFeed.log.info {{ event: :message_processed, metadata: event.metadata }}
```

Comment:
Yes, I spoke with @jaynefox about this, and as long as the first log entry has the nested JSON then logstash will handle it